src/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreeting } from './redux/actions';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { randomGreeting } = useSelector((state) => state.greeting.randomGreeting);
```

Comment:
* Kindly adjust your code here the variable {randomGreeting} shouldn't be nested instead you should write it this way  randomGreeting as I don't get the random greetings from the backend API but when i change the variable the greetings show up
![image](https://github.com/DJ-MrJay/hello-react-front-end/assets/52497246/21699a3c-379a-4ed6-9467-7f1b5bdd631f)
![image](https://github.com/DJ-MrJay/hello-react-front-end/assets/52497246/109ad9f7-799b-4057-895a-298e0e6be026)

Reply:
Thanks for the tip!